content/guides/exporters/supported-exporters/Go/Honeycomb.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Honeycomb.io (Tracing)"
+date: 2018-09-19
+draft: false
+weight: 3
+class: "resized-logo"
+aliases: [/supported-exporters/go/honeycomb]
+logo: /img/honeycomb-logo.jpg
+---
+
+- [Introduction](#introduction)
+- [Creating the exporter](#creating-the-exporter)
+- [Viewing your traces](#viewing-your-traces)
+- [Project link](#project-link)
+
+## Introduction
+
+Honeycomb is a saas tool that provides real-time system debugging. Visualize individual traces to deeply understand request execution. Break down, filter, and aggregate trace data to uncover patterns, find outliers, and understand historical trends.
```

Comment:
this is the blurb i'm suggesting we use elsewhere, feel free to cut and paste as it feels right: "Honeycomb is a hosted service for debugging your software in production. Capture traces or individual events, then speedily slice and dice your data to uncover patterns, find outliers, and understand historical trends."